daemons/controld/controld_messages.c

```diff
@@ -719,7 +719,7 @@ handle_lrm_delete(xmlNode *stored_msg)
         rsc_id = pcmk__xe_id(rsc_xml);
         from_sys = crm_element_value(stored_msg, PCMK__XA_CRM_SYS_FROM);
         node = crm_element_value(msg_data, PCMK__META_ON_NODE);
-        user_name = pcmk__update_acl_user(stored_msg, PCMK__XA_CRM_USER, NULL);
```

Comment:
We can drop the `PCMK__XA_CRM_USER` constant

Reply:
thanks